ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml

```diff
@@ -132,25 +132,6 @@ tests:
     env:
       TEST_SUITE: full
     workflow: assisted-baremetal
-- as: e2e-metal-assisted-tpmv2
```

Comment:
Maybe leave the presubmit one, so that we can test bug-fix PRs

Reply:
OK